ports/gtsam/portfile.cmake

```
@@ -2,13 +2,18 @@ vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO borglab/gtsam
     REF ${VERSION}
-    SHA512 0aae0b785a3f7ae25008d0938848e93519786521cca9cd0cd1a8937ec5ac46f3b1ca1bfaaff1ca5812c92f8ef55b729a06c57632da5dd8fc38afc22d3047f8e0
+    SHA512 c0e5de8d86ea8241b49449bd291999ec0d6530bc9943b213e7c650b0ab29894ab53636bd1a0ed82d9d9d148dfc399ebff28e108b060d2d2176b584823bd722cd
     HEAD_REF master    
```

```suggestion
    HEAD_REF develop
```
The default branch of gtsam is `develop` instead of `master`.

---

yes, thank you for pointing that out